data/hai/hai reveal 4 middle.txt

```diff
@@ -1269,7 +1272,7 @@ mission "Hai Reveal [A12] Into the Lion's Den"
 mission "Hai Reveal [A13] A New Elder"
 	landing
 	name "A New Elder"
-	description "Travel with Teeneep and the diplomatic corps to <stopover> and escort the Unfettered to <planet>."
+	description "Travel with Teeneep and the diplomatic corps to <stopovers> and escort the Unfettered to <planet>."
```

Comment:
Did this break in-game?
The wiki does say it's explicitly `<stopovers>` so if it did this should be a wiki update.
![image](https://user-images.githubusercontent.com/60949828/197459115-e5789dd9-5839-40e5-b7cb-a3d42496be23.png)

Reply:
<stopover> just shows as <stopover> in game indeed, it needs to be <stopover**s**>